plugins/cue_points/base/batch/copyCuePoints/Engine/KCopyCuePointEngine.php

```diff
@@ -7,13 +7,24 @@ abstract class KCopyCuePointEngine
 {
 	const MAX_CUE_POINT_CHUNKS = 500;
 
+	const CUE_POINT_THUMB = 'thumbCuePoint.Thumb';
+	const CUE_POINT_EVENT = 'eventCuePoint.Event';
+	const ANNOTATION = 'annotation.Annotation';
+	const CUE_POINT_AD = 'adCuePoint.Ad';
+	const CUE_POINT_CODE = 'codeCuePoint.Code';
+
 	protected $data = null;
 	protected $partnerId = null;
 	private $lastCuePointPerType = null;
 
 	abstract public function copyCuePoints();
 
-	protected function shouldCopyCuePoint($cuePoint) {return true;}
+	protected function shouldCopyCuePoint($cuePoint)
+	{
+		if ($cuePoint->cuePointType == self::CUE_POINT_CODE && in_array("poll-data", explode(",", $cuePoint->tags)))
```

Comment:
Please create array - that will hold "poll-data" or any other future extention per cue point type , and just check if it exist ./
Array ( key (cuepoint_Type _ value (filed , expected value))
call it extendedShouldCopyArray